Apps/Contoso.UWP.Puppet/MainPage.xaml.cs

```diff
@@ -35,5 +39,22 @@ private void ThrowException(object sender, RoutedEventArgs e)
         {
             throw new Exception();
         }
+
+        private void HandleMemoryWarning(object sender, RoutedEventArgs e)
+        {
+            List<byte[]> buffer = new List<byte[]>();
+            try
+            {
+                while (true)
+                {
+                    int blockSize = 128 * 1024 * 1024;
+                    buffer.Add(new byte[blockSize]);
+                    System.Diagnostics.Debug.WriteLine(String.Format("Memory allocated: {0} bytes", (blockSize * buffer.Count)));
+                }
+            }
+            catch (OutOfMemoryException ignore)
```

Comment:
Why do you try-catch OOM here? It should be okay to get the app crash.

Reply:
To get the time to handle memory warning I guess. Another option here is using some timers instead of just infinite loop

Reply:
Sounds like it doesn't catch the memory warning without this try-catch or timer.
We shouldn't do any other logic in test app to make SDK work. To me it feels like we couldn't catch low memory warning at all.